about.html

```diff
@@ -0,0 +1,137 @@
+                 <li class="nav-links-about"><a href="index.html">Home</a></li>
+                 <li class="nav-links-about"><a href="#">Exhibition</a></li>
+                 <li class="nav-links-about"><a href="#">Join</a></li>
+                 <li class="nav-links-about"><a href="#">Sponsor</a></li>
+                 <li  class="nav-links-about"><a href="#">Schedules</a></li>
+                 <li class="nav-links-about"><a class="border-link" href="#program">PROGRAMS</a></li>
+             </ul>
+     </nav>
+    </header>
+  
+    <article>
+        <!--*******Mobile Headline Section******-->
+<div class="headline-about">
+    <h3 class="topheadingh3-about">Hello Tech World!</h3>
+    <div class="adjustment">
+        <h1 class="topheadingh1-about">Web Global Annual Summit 2023</h1>
+    </div>
+    <div class="paradiv-about"><p class="headlinepara-about">What is Web Summit? <br><br>
+            
+        At a time of great uncertainty for many industries and, indeed, the world itself, we gather policymakers,
+         heads of state, and the founders and CEOs of technology companies and fast-growing startups, to ask a simple 
+         question: Where to next?<br><br>
+
+        See you in Lisbon this November.
+    </p></div>
+    <div class="datediv-about">
+        <p class="locationpara-about">
+            Please contact us per Email for further questions about Web Summit 2023!
+        </p>
+        <h3 class="dateheading-about underline">baqar2023@webSummitProject.org</h3>
+        
+      </div>
+</div>
+<!--*******Desktop Headline Section******-->
+<div class="headlinedesktop-about">
+    <div class="topdesktop-about">
+        <h3 class="topheadingdesktop-about">"Hello Tech World!"</h3>
+    </div>
+    <div class="topheadingdesktopdiv-about">
+        <h1 class="topheadingh1desktop-about">Web Global Annual <br> Summit  2023</h1>
+        <div class="paradivdesktop-about"><p class="headlineparadesktop-about">What is Web Summit? <br><br>
+            Politico has said we run “the world’s premier tech conference”,
+            the Atlantic that Web Summit is “where the future goes to be born”, and the New York Times that we assemble “a grand 
+            conclave of the tech industry’s high priests”. <br>    
+            At a time of great uncertainty for many industries and, indeed, the world itself, we gather policymakers,
+             heads of state, and the founders and CEOs of technology companies and fast-growing startups, to ask a simple 
+             question: Where to next?<br><br>
+    
+            See you in Lisbon this November.
+        </p></div>
+        <div class="datedivdesktop-about">
+            <p class="locationparadesktop-about">
+                Please contact us per Email for further questions about Web Summit 2023!
+            </p>
+            <h3 class="dateheadingdesktop-about underline">baqar2023@webSummitProject.org</h3>
```

Comment:
- [ ] Kindly use a `section` tag here also. i suggest you make reference to my detailed review online `53` above 👍

Reply:
Suggested changes have been incorporated and section tag added to this part of the code. Code reference mentioned below for your kind reference.

<section>
        <div class="headline-about">
    <h3 class="topheadingh3-about">Hello Tech World!</h3>
    <div class="adjustment">
        <h1 class="topheadingh1-about">Web Global Annual Summit 2023</h1>
    </div>
    <div class="paradiv-about"><p class="headlinepara-about">What is Web Summit? <br><br>
            
        At a time of great uncertainty for many industries and, indeed, the world itself, we gather policymakers,
         heads of state, and the founders and CEOs of technology companies and fast-growing startups, to ask a simple 
         question: Where to next?<br><br>

        See you in Lisbon this November.
    </p></div>
    <div class="datediv-about">
        <p class="locationpara-about">
            Please contact us per Email for further questions about Web Summit 2023!
        </p>
        <h3 class="dateheading-about underline">baqar2023@webSummitProject.org</h3>
        
      </div>
</div>
</section>